### DISCUSSION TITLE
I've been selected to be a GitHub Global Campus Student.

### DISCUSSION THREAD

**Discussion by primado** [community/community#36186] [GitHub Education]
Hello fellow teammates, I've been selected as a GitHub Global Campus Student lead in my school. I'm so happy to be here.
I'm open to connecting and collaborating. 
Thanks, I'm really excited to be here.

**Comment by radar-blip** [comment]
Good going and congratulations! 👍🏻

**Reply by primado** [reply]
Thank you

**Reply by radar-blip** [reply]
Anytime.

**Reply by 3kh0** [reply]
Great job @primado!
